src/uct/cuda/cuda_ipc/cuda_ipc_md.c

```diff
@@ -123,9 +123,8 @@ ucs_status_t uct_cuda_ipc_get_unique_index_for_uuid(int* idx,
             return UCS_ERR_NO_MEMORY;
         }
 
-        uct_cuda_ipc_memset(md->peer_accessible_cache + original_cache_size,
-                            UCS_TRY, new_capacity - original_capacity,
-                            sizeof(ucs_ternary_value_t));
+        uct_cuda_ipc_accessible_cache_init(md->peer_accessible_cache + original_cache_size,
+                                           new_capacity - original_capacity);
```

Comment:
i think we don't need this as a separate function, can just expand the code here